src/logger/POSTRequestLogger.ts

```diff
@@ -0,0 +1,127 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import { Logger, LogLevel } from '../index';
+import IntervalScheduler from '../scheduler/IntervalScheduler';
+
+export default class POSTRequestLogger implements Logger {
+  private logCapture: string[] = [];
+  private logSequenceNumber: number = 0;
+  private lock = false;
+  private static BATCH_SIZES = 75;
+  private intervalScheduler: IntervalScheduler;
+
+  constructor(
+    private name: string,
+    private meetingId: string,
+    private attendeeId: string,
```

Comment:
The logger should be independent of the meeting session. Meeting and attendee IDs are scoped to meeting sessions. It is the responsibility of the thing doing the logging to ensure that the meeting ID and attendee ID are part of the log line. Suggest removing these.

Reply:
Should we pass  MeetingSessionConfiguration object as a parameter and then read the meetingId and the AttendeeId ?

Reply:
The logger should be independent of the meeting session. It should not care about meeting sessions or their properties.